docker/README.md

```diff
@@ -75,7 +75,7 @@ distribution using debians.
    image of Gazebo Garden:
 
     ```
-    ./build.bash gz ./Dockerfile.gz
+    ./build.bash gz-ionic ./Dockerfile.gz
```

Comment:
Okay, but now we need to update the instruction right above to say "Gazebo Ionic"

Reply:
I'll roll-back it here to garden and lines below too, due to `build.bash` not working properly with Ionic AFAIU. It executes `Dockerfile.gz`, which is based on `nvidia/opengl:1.2-glvnd-devel-ubuntu20.04` image.